spec/services/search_spec.rb

```diff
@@ -86,39 +75,24 @@
       let!(:multi_match_page) {
         create(:page,
                title: 'multimatch page',
-               widgets: [matching_widget, petition_widget],
                language: language,
                campaign: campaign,
                tags: [tag])
       }
-      let(:new_params) {
+      let(:multi_match_params) {
         { search: {
-            content_search: test_text,
+            content_search: 'multimatch page',
             tags: [tag.id],
-            widget_type: ['PetitionWidget'],
             language: language.id
           }
         }
       }
-      let(:new_page_searcher) { Search::PageSearcher.new(new_params) }
-      it 'finds a page that matches the search query by tags, language, widget type and text content' do
+      let(:new_page_searcher) { Search::PageSearcher.new(multi_match_params) }
+      it 'finds a page that matches the search query by tags, language and text content' do
         expect(new_page_searcher.search).to match_array([multi_match_page])
       end
     end
 
   end
 
-  describe 'WidgetSearcher' do
-
-    it 'finds the matching widget based on the search text' do
-      expect(Search::WidgetSearcher.text_widget_search(test_text)).to eq([matching_widget])
-    end
-
-    it 'finds widgets that match the specified widget type' do
-      expect(Search::WidgetSearcher.widget_type_search(['PetitionWidget'])).to match_array(
-                [petition_widget])
-    end
-
-  end
-
 end
```

Comment:
Search by title and search by text aren't tested separately because they are private methods for content search. I was testing them separately before but then got feedback from Omar that it makes sense to make them private. We then talked about how to test them, and agreed that the way to go would be to just test the page searcher with different sets of parameters. 

I agree that the specs could be beefier, but testing the methods is a little implicit because they're private. I'm not sure where to draw the line on over-testing private methods.

As to the test for search by multiple criteria - I can add some more cases for that.